.flake8

```diff
@@ -60,7 +60,7 @@ per-file-ignores =
   # WPS412 logic of an extension is in __init__.py file;
   # FIXME: WPS201 too many imports
   # FIXME: WPS402 too many `noqa`s
-  src/sphinxcontrib/towncrier/__init__.py: E800, WPS201, WPS402, WPS412
+  src/sphinxcontrib/towncrier/__init__.py: E800, WPS201, WPS402, WPS412, WPS202
```

Comment:
Usually, it's a bad idea to add ignores w/o a compelling reason. Disabling the rule that makes sure that the module is not too complicated is not something acceptable. Besides, this rule can be configured to allow more module members.

I'd prefer for the helpers to just move out of the module that's getting complex...